README.md

```diff
@@ -1,6 +1,6 @@
 # docker-tippecanoe
 
-[![CircleCI](https://circleci.com/gh/azavea/docker-tippecanoe.svg?style=svg)](https://circleci.com/gh/azavea/docker-tippecanoe)
+![](https://github.com/azavea/docker-tippecanoe/workflows/CI/badge.svg)
 
 This repository contains a collection of templated `Dockerfile` for image variants designed to support the usage of [Tippecanoe](https://github.com/mapbox/tippecanoe).
```

Comment:
```diff
diff --git a/README.md b/README.md
index aa9893d..dd8f337 100644
--- a/README.md
+++ b/README.md
@@ -2,14 +2,14 @@
 
 ![](https://github.com/azavea/docker-tippecanoe/workflows/CI/badge.svg)
 
-This repository contains a collection of templated `Dockerfile` for image variants designed to support the usage of [Tippecanoe](https://github.com/mapbox/tippecanoe).
+This repository contains a templated `Dockerfile` designed to support the usage of multiple [Tippecanoe](https://github.com/mapbox/tippecanoe) versions.
 
 ## Usage
 
 ### Template Variables
 
 - `TIPPECANOE_VERSION` - Tippecanoe version
-- `VARIANT` - Base container image variant
+- `VARIANT` - Base container image variant. Currently supports: `slim`.
 
 ### Testing
 
```

Here's a tweak to the README language that clarifies variant support and repo intent.

Reply:
Addressed in 794b8d945b7bcbbc8c2a115af3a499335a6483c2.